packages/gatsby/src/utils/webpack.config.js

```diff
@@ -29,6 +30,18 @@ module.exports = async (
 ) => {
   const directoryPath = withBasePath(directory)
 
+  // Polyfill for IE support
+  const supportedBrowsers = browserslist(program.browserslist)
+  let iePolyfill = false
+  if (
+    supportedBrowsers.includes(`ie 9`) ||
+    supportedBrowsers.includes(`ie 10`)
```

Comment:
there are `ie-mob 10` and `ie-mob 11`, should we also check for those versions? Personally I don't want to do that